### PR TITLE
[3.13] Fix typos in docs and what's new

### PR DESCRIPTION
**Pull request by Wulian233**
cherry pick https://github.com/python/cpython/pull/123336

Because conflict https://github.com/python/cpython/pull/123336#issuecomment-2315102493

I'm not going to create a PR for 3.12
https://github.com/python/cpython/pull/123336#issuecomment-2315122695 @hugovk 

----
📚 Documentation preview 📚: https://cpython-previews--123451.org.readthedocs.build/

### CODE REVIEW

File: Lib/_pyrepl/completing_reader.py [hunk 1]
```diff
@@ -169,7 +169,7 @@ def do(self) -> None:
         r = self.reader  # type: ignore[assignment]
         last_is_completer = r.last_command_is(self.__class__)
         immutable_completions = r.assume_immutable_completions
-        completions_unchangable = last_is_completer and immutable_completions
+        completions_unchangeable = last_is_completer and immutable_completions
         stem = r.get_stem()
         if not completions_unchangable:
```
Comment (AlexWaygood) [inline Lib/_pyrepl/completing_reader.py]: Won't this now result in a NameError?

Comment (AlexWaygood) [reply Lib/_pyrepl/completing_reader.py]: (The answer is "yes", and it's causing several tests to fail.)

You say that this PR is backporting https://github.com/python/cpython/pull/123336, but I don't see any changes to this file in https://github.com/python/cpython/pull/123336

**Review by AlexWaygood — DISMISSED** [review]
This backport PR seems to be making changes the original PR didn't make, which are incorrect and are causing test failures

File: Lib/_pyrepl/completing_reader.py (hunk 1 above)
Comment (Wulian233) [reply Lib/_pyrepl/completing_reader.py]: 😨I made a mistake, I didn't notice the replacement when I made the change

**Comment by Wulian233** [comment]
Also contain https://github.com/python/cpython/pull/123201

**Comment by encukou** [comment]
As a reminder, 3.13 is in RC phase; code changes need a *very* good reason to be included.

Could you limit this to the documentation?

**Comment by Wulian233** [comment]
These are just comments in the code, not real changes. But if necessary, I can limit docs and what's new parts,  total 4 files :)

**Comment by Wulian233** [comment]
Now documents(error message) only

File: Modules/_testcapi/run.c [hunk 2]
```diff
@@ -78,14 +78,14 @@ run_fileexflags(PyObject *mod, PyObject *pos_args)
     result = PyRun_FileExFlags(fp, filename, start, globals, locals, closeit, pflags);
 
     if (closeit && result && _Py_IsValidFD(fd)) {
-        PyErr_SetString(PyExc_AssertionError, "File was not closed after excution");
+        PyErr_SetString(PyExc_AssertionError, "File was not closed after execution");
```
Comment (encukou) [inline Modules/_testcapi/run.c]: This error message in a test module is not user-visible; it's not necessary to change it.

Comment (Wulian233) [reply Modules/_testcapi/run.c]: revolved

**Review by hugovk — APPROVED** [review]

**Pull request merged**